src/Classes/ListingHydrator.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+
+namespace ListingsApp\Classes;
+require_once '../../vendor/autoload.php';
+
+use PDO;
+
+/** connects to database
+ * @return PDO
+ */
+
+function getDB(): PDO
+{
+    return new PDO('mysql:host=db;dbname=listings-feed', 'root', 'password');
+
+}
```

Comment:
Whats this doing in here? Only one class/thing per file please!

Reply:
We were testing it and forgot to get it out :-/